torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1894,26 +1895,20 @@ def sample_inputs_t(op_info, device, dtype, requires_grad, **kwargs):
 
 
 def sample_inputs_mm(op_info, device, dtype, requires_grad, **kwargs):
-    make_arg = partial(make_tensor, device=device, dtype=dtype)
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
 
-    first_shape, second_shape = (S, M), (M, S)
-    sample_inputs = []
+    def make_arg_conj(size):
```

Comment:
Wait what does this function do? Doesn't it need a conj()?

Reply:
Follow-up question for @anjali411 -- should sample inputs be generating conjugated samples, or are those automatically generated now?

Reply:
Oh, sorry, I completely missed the `conj()`! Thanks for the catch.

Once we test conjugated inputs on all the possible inputs (similar to the extensions we did for `noncontiguous`), these samples will not be necessary.

Reply:
@mruberry they are generated automatically for all relevant tests I think. 

I added this logic manually to generate conjugated samples when required:

```
include_conjugated_inputs = op.test_conjugated_samples and dtype.is_complex
        samples = op.sample_inputs(device, dtype, requires_grad=_requires_grad, include_conjugated_inputs=include_conjugated_inputs)
```

I see that we are not generating conjugated samples in `test_noncontiguous_samples`!